src/main/resources/com/google/api/codegen/go/sample.snip

```diff
@@ -14,6 +14,8 @@
     //    If not already done, install the gcloud CLI from
     //    https://cloud.google.com/sdk/ and run
     //    `gcloud beta auth application-default login`
+    //    For more information, see:
```

Comment:
It's a fair point --- I think it would be fine to remove all the colons. We use them, for instance, to refer to external URLs for media upload/download and auth instructions.

It would be nice if we could get API explorer to hyperlink URLs appearing in our samples.